html/captive-portal/templates/signin.html

```diff
@@ -8,6 +8,11 @@
     <div class="c-card u-padding u-padding@tablet u-padding-top-none">
         [% form.get_field('aup').render | none %]
     </div>
+
+      [% IF aup_only %]
+      <script src="/common/aup-autosubmit.js">
```

Comment:
`<script>` is not an empty tag and should be closed with an end tag.

Reply:
As they say in France: `fixaide`